docs/ref/forms/fields.txt

```diff
@@ -1142,6 +1142,10 @@ method::
             super().__init__(*args, **kwargs)
             self.fields['foo_select'].queryset = ...
 
+Both fields have the attribute ``iterator`` which specifies the class used to
```

Comment:
Both ``ModelChoiceField`` and ``ModelMultipleChoiceField`` have an ``iterator`` attribute... 

(Too far from the previous reference for just "both fields".)

I think we need to link to a bigger discussion, perhaps from a paragraph here. (See main comment.)